GVRf/Framework/framework/src/main/jni/engine/renderer/renderer.cpp

```diff
@@ -348,14 +348,17 @@ void addRenderData(RenderData *render_data) {
 void Renderer::occlusion_cull(Scene* scene,
         std::vector<SceneObject*>& scene_objects, ShaderManager *shader_manager,
         glm::mat4 vp_matrix) {
-
+    scene->lockColliders();
```

Comment:
Is it okay to have the overhead of performing a lock/unlock and pick check if there are no colliders or pickers in the scene?

Just curious since this is the renderer's code.